### PR TITLE
Raspbian config

### PR DESCRIPTION
**Pull request by julibert**
This PR adds Raspbian configuration to the build system.

### CODE REVIEW

File: micro_ros_setup/config/raspbian/generic/configure.sh [hunk 1]
```diff
@@ -0,0 +1,26 @@
+#! /bin/bash
+
+pushd $FW_TARGETDIR >/dev/null
+    rm -rf mcu_ws/*
+    cp raspbian_apps/toolchain.cmake mcu_ws/
```
Comment (pablogs9) [inline micro_ros_setup/config/raspbian/generic/configure.sh:5]: Why this is also in create step?

File: micro_ros_setup/config/raspbian/generic/configure.sh [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    curl -s https://raw.githubusercontent.com/ros2/ros2/dashing/ros2.repos |\
+        ros2 run micro_ros_setup yaml_filter.py raspbian_apps/$CONFIG_NAME/ros2_repos.filter > ros2.repos
+    vcs import --input ros2.repos mcu_ws/ && rm ros2.repos
+    if [ -d mcu_ws/ros2/rcl_logging/rcl_logging_log4cxx ]; then
+        touch mcu_ws/ros2/rcl_logging/rcl_logging_log4cxx/COLCON_IGNORE
+    fi
+    vcs import --input raspbian_apps/$CONFIG_NAME/app.repos mcu_ws/
+    if [ -d raspbian_apps/$CONFIG_NAME/app ]; then
+        cp -r raspbian_apps/$CONFIG_NAME/app mcu_ws/
+    fi
```
Comment (pablogs9) [inline micro_ros_setup/config/raspbian/generic/configure.sh:15]: Why are these downloading packages steps in configuration?

Comment (julibert) [reply micro_ros_setup/config/raspbian/generic/configure.sh:15]: Why not? You know the target app in configuration so you are able to download the packages needed.

File: micro_ros_setup/config/raspbian/generic/configure.sh (hunk 1 above)
Comment (julibert) [reply micro_ros_setup/config/raspbian/generic/configure.sh:5]: It was a mistake. I removed it.

**Comment by pablogs9** [comment]
It requires commits to be signed

**Pull request merged**